app/forms/income/TurnoverIncomeAmountFormProvider.scala

```diff
@@ -14,19 +14,20 @@
  * limitations under the License.
  */
 
-package forms
+package forms.income
 
 import forms.mappings.Mappings
-import javax.inject.Inject
 import play.api.data.Form
 
+import javax.inject.Inject
+
 class TurnoverIncomeAmountFormProvider @Inject() extends Mappings {
```

Comment:
Should this form have another validation for when `the amount of tax entered is higher than the amount of income previously entered`? Or will this be done in another story?

Reply:
Not necessary for the current prototype 6.2.5 these pages are based on